tipg/database.py

```diff
@@ -59,12 +61,13 @@ async def __call__(self, conn: asyncpg.Connection):
             """
         )
 
-        # Register custom SQL functions/table/views in pg_temp
-        for sqlfile in self.user_sql_files:
-            await conn.execute(sqlfile.read_text())
+        if database_settings.write_functions:
+            # Register custom SQL functions/table/views in pg_temp
+            for sqlfile in self.user_sql_files:
+                await conn.execute(sqlfile.read_text())
```

Comment:
I'm not quite sure we need this 🤷 we will register (in pg_temp) functions that are passed by the user so having both options is kinda weird to me